### PR TITLE
Fixes #15079: Soften dependencies in nightly builds

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15079

### CODE REVIEW

File: rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec [hunk 1]
```diff
@@ -60,7 +60,7 @@ BuildArch: noarch
 
 # Dependencies
 
-Requires: rudder-inventory-ldap = %{real_epoch}:%{real_version}, rudder-jetty = %{real_epoch}:%{real_version},
+Requires: %(../format-dependencies rpm %{real_epoch}:%{real_version} rudder-inventory-ldap rudder-jetty)
```
Comment (amousset) [inline rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec]: where does this script come from?

Comment (peckpeck) [reply rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec]: oops

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**